scatter.cfg

```diff
@@ -0,0 +1,35 @@
+[#]
+sets = ["enso_diags"]
+plot_type = "scatter"
+case_id = "ocean-zonal-wind-stress"
```

Comment:
change `TAUX-feedback` so that they are consistent with the *.cfg files for regression coefficient one.